ee/tabby-webserver/src/schema/mod.rs

```diff
@@ -21,9 +24,8 @@ use tracing::error;
 use validator::ValidationErrors;
 use worker::{Worker, WorkerService};
 
-use crate::schema::auth::{OAuthCredential, OAuthProvider};
-
 pub trait ServiceLocator: Send + Sync {
+    fn https_client(&self) -> Client<HttpsConnector<HttpConnector>>;
```

Comment:
as suggested above, I feel this client class should be a member of state struct specific to oauth route, instead of puting inside service locator (which is specific for graphql apis)